library/pod-security-policy/seccomp/src.rego

```diff
@@ -0,0 +1,20 @@
+package k8spspseccomp
+
+violation[{"msg": msg, "details": {}}] {
+    metadata := input.review.object.metadata
+    not input_seccomp_allowed(metadata)
+    msg := sprintf("Seccomp profile is not allowed, pod: %v. Allowed profiles: %v", [input.review.object.metadata.name, input.parameters.allowedProfiles])
+}
+
+input_seccomp_allowed(metadata) {
+    metadata.annotations
+    not metadata.annotations["seccomp.security.alpha.kubernetes.io/pod"]
```

Comment:
we should look at this too `container.seccomp.security.alpha.kubernetes.io/${container_name}`
for example: https://github.com/kubernetes/kubernetes/blob/release-1.4/docs/design/seccomp.md#examples

Reply:
thanks, added now